procedures/release-duty/desktop/esr-to-esr-updates.rst

```diff
@@ -83,3 +83,35 @@ have been superceded by the new ``esr*`` rules.
 .. |platform-deprecation| image:: /procedures/release-duty/desktop/platform-deprecation.png
 .. |watershed| image:: /procedures/release-duty/desktop/watershed.png
 
+
+The Snap case
+-------------
+
+In addition to Balrog, you need to update the Snap store too. For more information about Snap,
+see ``procedures/release-duty/desktop/ubuntu-snap.rst``. Steps:
```

Comment:
Nit, not blocking: did we want to href here?

Reply:
Yeah, my rst skills aren't great 😅